using_openshift/master_node_configuration.adoc

```diff
@@ -0,0 +1,141 @@
+  kubernetesPublicURL: https://10.0.0.1:8443
+  logoutURI: ""
+  masterPublicURL: https://10.0.0.1:8443
+  publicURL: https://10.0.0.1:8444
+  servingInfo:
+    bindAddress: 0.0.0.0:8444
+    certFile: openshift.local.certificates/master/cert.crt
+    clientCA: openshift.local.certificates/ca/cert.crt
+    keyFile: openshift.local.certificates/master/key.key
+corsAllowedOrigins:
+- 10.0.0.1:8444
+- 10.0.0.1:8443
+- localhost
+- 127.0.0.1
+dnsConfig:
+  bindAddress: 0.0.0.0:53
+etcdClientInfo:
+  ca: ""
+  certFile: ""
+  keyFile: ""
+  url: http://10.0.0.1:4001
+etcdConfig:
+  masterAddress: 10.0.0.1:4001
+  peerAddress: 0.0.0.0:7001
+  servingInfo:
+    bindAddress: 0.0.0.0:4001
+    certFile: ""
+    clientCA: ""
+    keyFile: ""
+  storageDirectory: openshift.local.etcd
+imageConfig:
+  format: openshift/origin-${component}:${version}
+  latest: false
+kind: MasterConfig
+kubernetesMasterConfig:
+  masterIP: 10.0.0.1
+  servicesSubnet: 172.30.17.0/24
+  staticNodeNames:
+  - hostname.example.org
+masterClients:
+  deployerKubeConfig: openshift.local.certificates/openshift-deployer/.kubeconfig
+  kubernetesKubeConfig: openshift.local.certificates/kube-client/.kubeconfig
+  openshiftLoopbackKubeConfig: openshift.local.certificates/openshift-client/.kubeconfig
+oauthConfig:
+  assetPublicURL: https://10.0.0.1:8444
+  grantConfig:
+    method: auto
+  identityProviders:
+  - provider:
+      apiVersion: v1
+      kind: AllowAllPasswordIdentityProvider
+    usage:
+      challenge: true
+      login: true
+      providerScope: anypassword
+  masterPublicURL: https://10.0.0.1:8443
+  masterURL: https://10.0.0.1:8443
+  proxyCA: ""
+  sessionAuthenticationConfig:
+    sessionMaxAgeSeconds: 300
+    sessionName: ssn
+    sessionSecrets:
+    - e0646b24-d25e-11e4-8bfd-28d2447dc82b
+  tokenConfig:
+    accessTokenMaxAgeSeconds: 3600
+    authorizeTokenMaxAgeSeconds: 300
+policyConfig:
+  bootstrapPolicyFile: openshift.local.policy/policy.json
+  masterAuthorizationNamespace: master
+  openshiftSharedResourcesNamespace: openshift
+servingInfo:
+  bindAddress: 0.0.0.0:8443
+  certFile: openshift.local.certificates/master/cert.crt
+  clientCA: openshift.local.certificates/ca/cert.crt
+  keyFile: openshift.local.certificates/master/key.key
+
+---
+
+=== Stub for discussion of what the different values actually do
+
+== Example node.yaml
+This is an example at a point in time.  You should run `--write-config` to generate a file of your own.
+
+[source]
+---
+allowDisabledDocker: true
+apiVersion: v1
+dnsDomain: local
+dnsIP: 10.0.0.1
+kind: NodeConfig
+masterKubeConfig: openshift.local.certificates/node-hostname.example.org/.kubeconfig
+networkContainerImage: openshift/origin-pod:v0.4.1
+nodeName: hostname.example.org
+recordEvents: false
+servingInfo:
+  bindAddress: 0.0.0.0:10250
+  certFile: openshift.local.certificates/node-hostname.example.org/server.crt
+  clientCA: ""
+  keyFile: openshift.local.certificates/node-hostname.example.org/server.key
```

Comment:
When describing values, probably worth mentioning why these are not inlined... we want admins to be able to post these config files for help without having to redact sensitive stuff (urls and IPs may still be sensitive, but externalizing those seems excessive).

Just an observation, no need to hold up the PR